.devcontainer/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/devcontainers/rust:1-1-bookworm
+FROM mcr.microsoft.com/devcontainers/rust:1.0.5-1-bookworm
```

Alternatively, pin the the MSRV 1.70.0. See https://github.com/devcontainers/images/blob/main/src/rust/history/1.0.1.md#variant-bookworm
```suggestion
FROM mcr.microsoft.com/devcontainers/rust:1.0.1-1-bookworm
```